### PR TITLE
Improve style guide

### PR DESCRIPTION
**Pull request by nwithan8**
# Description

- Change style guide to compress dictionary/list initializers
- Apply new styling rules

# Testing

- No impact on functionality, tests

# Pull Request Type

Please select the option(s) that are relevant to this PR.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Improvement (fixing a typo, updating readme, renaming a variable name, etc)

### CODE REVIEW

File: EasyPost.Tests/InsuranceTest.cs [hunk 1]
```diff
@@ -34,12 +34,7 @@ public async Task TestAll()
         {
             UseVCR("all");
 
-            InsuranceCollection insuranceCollection = await Client.Insurance.All(new Dictionary<string, object>
-            {
-                {
-                    "page_size", Fixture.PageSize
-                }
-            });
+            InsuranceCollection insuranceCollection = await Client.Insurance.All(new Dictionary<string, object> { { "page_size", Fixture.PageSize } });
```
Comment (Justintime50) [inline EasyPost.Tests/InsuranceTest.cs:37]: Question: So this is where I was wondering if we could get the best of both worlds where nested dicts break out and indent but individual lines don't:
```suggestion
            InsuranceCollection insuranceCollection = await Client.Insurance.All(new Dictionary<string, object> {
                { "page_size", Fixture.PageSize }
            });
```

Comment (nwithan8) [reply EasyPost.Tests/InsuranceTest.cs:37]: Correct:
https://www.jetbrains.com/help/resharper/EditorConfig_CSHARP_LineBreaksPageSchema.html#resharper_csharp_place_simple_initializer_on_single_line

Unfortunately, if there's only one item in the dictionary/list, it won't "chop" to a new line (note that setting 0 and 1 are the same): https://www.jetbrains.com/help/resharper/EditorConfig_CSHARP_LineBreaksPageSchema.html#resharper_csharp_max_initializer_elements_on_line

Comment (Justintime50) [reply EasyPost.Tests/InsuranceTest.cs:37]: Ah ok, I see some other examples where multiple line items do get broken up. I can live with that.

Comment (nwithan8) [reply EasyPost.Tests/InsuranceTest.cs:37]: Can I get a stamp?

**Review by Justintime50 — APPROVED** [review]

File: EasyPost.Tests/InsuranceTest.cs (hunk 1 above)
Comment (Justintime50) [reply EasyPost.Tests/InsuranceTest.cs:37]: I... did?

**Review by jchen293 — APPROVED** [review]

**Pull request merged**